nautilus_core/model/src/data/greeks.rs

```diff
@@ -225,7 +225,7 @@ fn test_imply_vol_and_greeks_accuracy_call() {
     let tolerance = 1e-5;
     assert!(
         (implied_result.vol - sigma).abs() < tolerance,
-        "Implied volatility difference exceeds tolerance"
+        "Vol difference exceeds tolerance"
```

Comment:
Probably fine, but I always find "vol" vague as it can mean several different things (in this context fairly obvious though).